config.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
@Matteo04052017 This config file, should I make a copy of it and place it in the `charts/templates/data/` directory  or just  move it there?

Reply:
Nice question. For the moment, move it in the charts/tango-example/templates/data/ and adjust the docker-compose file for the dsconfig container. 

We'll talk about it tomorrow with @piersharding